tailwind.config.js

```diff
@@ -2,7 +2,48 @@
 module.exports = {
   content: ["./*.html", "./src/*.html"],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        urbanist: ["Urbanist", "sans-serif"],
+      },
+      backgroundImage: {
+        "linear-colors":
+          "linear-gradient(134.79deg, rgba(112, 59, 247, 0.5) -0.01%, rgba(112, 59, 247, 0) 9.65%)",
+      },
+      backgroundImage: {
+        "linear-gradient":
+          "linear-gradient(95.93deg, #262626 -14.82%, rgba(38, 38, 38, 0) 40.46%)",
+      },
+      backgroundImage: {
+        "img-for-header": "url('./assets/homepage/abstract-design.png')",
+      },
+      backgroundColor: {
+        "header-bg": "rgba(26, 26, 26, 1)",
```

Comment:
change to just header, it will resolve to bg-header

Reply:
changed